data/json/items/archery.json

```diff
@@ -887,7 +935,7 @@
       },
       "ranged_damage" : 15,
       "range" : 18,
-      "dispersion" : 120,
+      "dispersion" : 75,
```

Comment:
Reverted this and set the recurve to 120 based on IRC discussion.  Will need another rebalancing at a later date, but bows that claim to sacrifice accuracy for power should not be the most accurate ones in the game.